examples/demo-nft-module/tests/nft_test.rs

```diff
@@ -70,7 +76,11 @@ fn transfer() {
         owners: vec![(0, admin), (1, owner1), (2, owner2)],
     };
     let tmpdir = tempfile::tempdir().unwrap();
-    let mut working_set = WorkingSet::new(ProverStorage::with_path(tmpdir.path()).unwrap());
+    let mut working_set = WorkingSet::new(
+        todo!(),
```

Comment:
Let's put `gas & gas_price` in a single struct so the `WorkingSet::new` takes less arguments.